### ISSUE TITLE
@deprecated  fields in input type objects are not documented

### ISSUE THREAD

**Issue by sahingoko** [anvilco/spectaql#827]
Hi,
We are using @deprecated directive to mark field deprecation in our schema. And those are correctly documented with spectaql for object types other than input. With input objects, fields that are marked with the @deprecated directive is not documented at all, although schema is working fine and accepting those input fields. Is that intentional?

**Comment by newhouse** [comment]
@sahingoko this is now supported in `2.1.0` that I just published.

Because it's sort of a "breaking" change, it's not enabled by default. You'll need to set the [`introspection.inputValueDeprecation` option to `true` in your config file](https://github.com/anvilco/spectaql/blob/main/config-example.yml#L171), but then it should work.

**Comment by sahingoko** [comment]
@newhouse Yes, I just tested it in test environment and looks like it is working fine. Thanks again.
